rtengine/dcraw.c

```diff
@@ -3826,7 +3826,7 @@ void CLASS remove_zeroes()
 }
 
 /*
-   Seach from the current directory up to the root looking for
+   Search from the current directory up to the root looking for
```

Comment:
@heckflosse what do we do about the changes to the dcraw* files?
Am I right that the changes to `dcraw.c` should be reverted, but the changes to `dcraw.cc` and to `dcraw.patch` can stay?